generators/entity/index.js

```diff
@@ -994,9 +994,22 @@ class EntityGenerator extends BaseBlueprintGenerator {
                         }
                     }
 
+                    relationship.otherEntityFieldName = 'id';
                     if (otherEntityName === 'user') {
                         relationship.otherEntityTableName = `${jhiTablePrefix}_user`;
                         hasUserField = true;
+                    } else if (otherEntityName === 'authority') {
+                        relationship.otherEntityTableName = `${jhiTablePrefix}_authority`;
+                        relationship.otherEntityFieldName = 'name';
+                        if (relationship.otherEntityField === 'id') {
+                            this.warning(
+                                chalk.yellow(
```

Comment:
Remove chalk here.
Warning already does that.
https://github.com/jhipster/generator-jhipster/blob/0e4b8dc74087603bc781ce2d9983251722af0314/generators/generator-base.js#L1598-L1599